ansible_collections/arista/avd/roles/eos_designs/docs/how-to/custom-templates.md

```diff
@@ -0,0 +1,64 @@
+<!--
+  ~ Copyright (c) 2023 Arista Networks, Inc.
+  ~ Use of this source code is governed by the Apache License 2.0
+  ~ that can be found in the LICENSE file.
+  -->
+
+# Custom Templates
+
+With Custom Templates for the `eos_designs` role, the user can create additional structured configuration variables that would otherwise not be created by the builtin templates.
```

Comment:
```suggestion
With Custom Templates for the `eos_designs` role, the user can create additional structured configuration variables that would otherwise not be created by the built-in templates.
```